examples/ml_pipeline_example.yaml

```
@@ -0,0 +1,117 @@
+# This file contains an AWPL description for a workflow running in Kubernetes pods.
+# 
+# Workflow:
+# 1. Fetch data and calculate an initial metric.
+# 2. Loop: run preprocessing tasks until metric > 0.9.
+# 3. Train a model using preprocessed data.
+# 4. Fetch multiple test sets.
+# 5. Map: evaluate the model on each in parallel.
+# 6. Aggregate results and calculate average.
+# 7. Branch: if average > 0.95 -> advanced task; else -> basic task.
+# 8. Finalization step (cleanup, output).
+
+
```

Illustrating the beginning of the yaml file
```suggestion
---
```